### DISCUSSION TITLE
[expo-router] Shared Routes and Normal Routes with same name

### DISCUSSION THREAD

**Discussion by the-devbear** [expo/expo#27605] [Archived: Questions]
So I currently have this structure:
```
src
└── app
    ├── (app)
    │   ├── (home)
    │   │   ├── _layout.tsx
    │   │   ├── index.tsx
    │   │   └── users
    │   │       └── [id].tsx
    │   ├── (profile)
    │   │   ├── _layout.tsx
    │   │   ├── profile.tsx
    │   │   ├── settings.tsx
    │   │   └── users
    │   │       └── [id].tsx
    │   ├── _layout.tsx
    │   └── misc
    │       ├── _layout.tsx
    │       └── index.tsx
    ├── (auth)
    │   ├── _layout.tsx
    │   ├── sign-in.tsx
    │   └── sign-up.tsx
    ├── +html.tsx
    ├── +not-found.tsx
    └── _layout.tsx
```
And I would like to use [Shared Routes](https://docs.expo.dev/router/advanced/shared-routes/) so I don't have to copy the code around all the time. `(home)`, `(profile)` and `misc` all should be tabs with their own Stack.
The route names should stay the same. 

And the Settings Page for example should be only accessible from the profiles tab.

Is this even possible? Or do I need to copy the code?

Thanks in advance for the help 🫶

**Comment by billvog** [comment]
Hey, have you figured this out? Have the same problem here.
Whenever I'm creating a (home,profile) folder inside (app), which is a tabs layout, I get new tabs like (home)/user/[id] and (profile)/user/[id]. Then, if I create a _layout.tsx inside (home,profile) I get a "Duplicate screens for route (home)" error.

**Reply by marklawlor** [reply]
This is either a bug or there is an issue and the error isn't clear enough. Either way you should create an issue with a minimal reproduction so that we can provide support.

**Reply by the-devbear** [reply]
Thanks @marklawlor, I will do that

**Reply by the-devbear** [reply]
Created the Issue https://github.com/expo/expo/issues/28556
